CONTRIBUTING.md

```diff
@@ -274,18 +274,21 @@ There is a rather little risk that the plugin which is compatible with `X - 1`
 and does **not** use any `X EAP/RC`-specific API
 turns out to be **not** compatible with stable `X` release of IDE.
 
-For instance:
+Version updates can be done automatically using `./gradlew updateIntellijVersions`
```

Comment:
Okay! last step that's left, is to wire up the newly-added task correctly in CircleCI config... note that the cron is still going to call the non-existent `updateEapBuildNumber` task :/